requirements/env_hdk.yml

```diff
@@ -17,7 +17,7 @@ dependencies:
   - psutil
   - openpyxl
   - xlrd
-  - sqlalchemy
+  - sqlalchemy>=1.4.0,<1.4.46
```

Comment:
Adding a minimal border here to keep this aligned with the rest of the environments.